### PR TITLE
View sighting

### PR DESCRIPTION
**Pull request by pascuad21**
Purpose: 
Create a basic View Sighting Screen

Changes:
Added ViewSightingStack.jsx
Added View Sighting Screen to the hamburger menu

Testing:
Tested on android 

Image:
image

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -29,6 +30,7 @@ export default function App() {
           {/* As new screens are made, put them here to be able to view them */}
           <Drawer.Screen name={screens.home} component={HomeStackScreen} />
           <Drawer.Screen name={screens.login} component={LoginStackScreen} />
+          <Drawer.Screen name={screens.view_sighting} component={ViewSightingStackScreen} />
```
Comment (dborg291) [inline App.js:33]: The repo is using camelCase for naming, this should be `viewSighting`

File: src/constants/screens.js [hunk 2]
```diff
@@ -2,4 +2,5 @@ export default {
   home: 'Home',
   login: 'Login',
   loading: 'Loading',
+  view_sighting: 'View Sighting',
```
Comment (dborg291) [inline src/constants/screens.js:5]: camelCase

**Review by dborg291 — CHANGES_REQUESTED** [review]

File: App.js (hunk 1 above)
Comment (pascuad21) [reply App.js]: Ok changed it

File: src/constants/screens.js (hunk 2 above)
Comment (pascuad21) [reply src/constants/screens.js]: Ok changed it

**Review by dborg291 — APPROVED** [review]

**Pull request merged**